packages/i/intl-tel-input.json

```
@@ -26,18 +26,15 @@
       {
         "basePath": "build",
         "files": [
-          "js/utils.js",
-          "js/intlTelInput*.js",
-          "img/flags*.png",
-          "css/intlTelInput.css"
+          "**/*"
```

```suggestion
          "css/*.css",
          "js/*.js",
          "js/i18n/{,*/}*.js",
          "img/*.@(webp|png)"
```
Please avoid using an unrestricted globstar as it can lead to including files on the CDN that are not supported.